amundsen_application/static/js/components/SearchPage/SearchBar/index.tsx

```diff
@@ -89,19 +89,20 @@ class SearchBar extends React.Component<SearchBarProps, SearchBarState> {
   render() {
     const subTextClass = `subtext ${this.state.optionalSubTextClass}`;
     return (
-      <div id="search-bar" className="col-xs-12">
+      <div id="search-bar" className="col-xs-12 col-md-offset-1 col-md-10">
         <form className="search-bar-form" onSubmit={ this.handleValueSubmit }>
             <button className="btn btn-flat-icon search-bar-button" type="submit">
               <img className="icon icon-search" />
             </button>
             <input
-             className="search-bar-input form-control"
-             value={ this.state.searchTerm }
-             onChange={ this.handleValueChange }
-             aria-label={ this.props.placeholder }
-             placeholder={ this.props.placeholder }
-             autoFocus={ true }
-             ref={ this.inputRef }
+              id="searchInput"
```

Comment:
Personally I'd prefer to go kebab-case on all css IDs and class names, but changing the search bar's ID will erase everyone's search suggestion history, which I think we should preserve.

Reply:
That's dependent on the `id`? But this input element also didn't have an `id` before -- can you elaborate?

Reply:
I removed the `id` in the first commit and added it back in the 3rd commit. Yes, the browser auto complete is based on matching `name` and `id` attributes. I also tried to restyle the auto complete with `input:-webkit-autofill` and the like, but that didn't seem to work.

Reply:
For the nav bar, we looked at a few other internal tools. Amundsen had a pretty high height overall, so we wanted to get more closely aligned there. I do think it would be worth getting more signal of what of the menu links are being used, but that will be dependant on having instrumentation around those items.

For the search bar, we wanted to increase it slightly, but we may still tweak that to get it to balance in the layout overall. Since it is int he primary action on the page, it needs to have visual prominence in the layout.

I am working on a spec sheet that will allow for consistent padding across all table/row elements (so search results and table columns).

Reply:
@danwom I think it's great to want to preserve our user's search suggestions, but long term I think should accept that we will change it eventually.

The reason I think we have to change it eventually anyway is that it doesn't feel right to have naming inconsistencies -- which could prevent applying really solid lint rules in the future -- for the sake of a feature that Amundsen didn't intentionally implement and that we've never promised our users. We can further discuss offline if you have strong thoughts, or raise it again when it comes time for shoring up our lint rules.

Reply:
I'll update the ID in a followup commit.